client/src/Components/Photo.jsx

```diff
@@ -73,8 +27,8 @@ class Photo extends React.Component {
     const { photo, index, isContainerHovered } = this.props;
     const { hover } = this.state;
     return (
-      <StyledPhoto index={index}>
-        <StyledImage
+      <StyledPhotoContainer index={index}>
+        <StyledPhoto
           src={photo.photo_url}
           alt="home interior"
```

Comment:
you can move this static text 'alt="home interior"' into the styled component since it doesn't seem to change?

Reply:
How would you do this? Can you give me an example?

Reply:
ahh...just kidding...i thought you could D: but i guess you can't set it in css. nvm!